repose-aggregator/components/filters/filter-bundle/pom.xml

```diff
@@ -204,6 +204,11 @@
             <artifactId>cors</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.openrepose</groupId>
+            <artifactId>ip-classification</artifactId>
```

Comment:
So do we want to change this one?
If so, then to what?
Just append user to it `ip-classification-user`?
Or do we leave it alone?